### ISSUE TITLE
Checking streams instead of hardcoded folders

### ISSUE THREAD

**Issue by bkostrowiecki** [getgrav/grav-plugin-problems#11]
Hi there,

I tried to use this plugin to check if I can use Grav CMS on my hosting. I found a problem with permissions of logs directory. I changed the stream for `logs` to a different folder, but sadly Problems Plugin still checks the hardcoded `logs` folder instead of new folder pointed by `logs` stream.

I can think of two possible solution.
1. Create a configuration file with paths
2. Using streams from Grav CMS instead of hardcoded paths

What do you think about this?

**Comment by rhukster** [comment]
Streams are not fully loaded when problems runs.  I’ll,check tho.

**Comment by jza34** [comment]
Hi,
May someone have a look at this issue because today I am in this situation with latest version of GRAV.
Hosting company is 1&1, the logs directory is not writeable by the owner...by concept. So the alternative stream grav-logs is working but no way to activate the "problem" plugin without being stuck to error 500.
Thanks guys for the good work so far!

**Comment by mahagr** [comment]
Just disable the plugin and you should be fine.

**Comment by bkostrowiecki** [comment]
Yeah, I did it, but I don't believe it's a best possible solution. I assume this plugin should help in checking server configuration, so false negative is rather annoying.

**Comment by mahagr** [comment]
@rhukster Streams should be fully loaded by plugin initialize, so I cannot see why it won't work.
